src/game.ts

```diff
@@ -346,37 +330,20 @@ export class Game {
         this.players.forEach((player) => {
             player.elem.style.left = player.posX + "px";
             player.elem.style.top = player.posY + "px";
+            if (player.blastCounter > 0) {
+                player.blastCounter -= 1;
+            }
         });
     }
 
-    private updateBlasts(blast: Blast, num: number) {
-        if (blast.opacity > 0) {
-            // reduce blast opacity
-            blast.posX -= this.config.playerSize / 6;
-            blast.posY -= this.config.playerSize / 6;
-            blast.size += this.config.playerSize / 3;
-            blast.opacity -= 0.05;
-            blast.elem.style.opacity = blast.opacity.toString();
-            blast.elem.style.left = blast.posX + "px";
-            blast.elem.style.top = blast.posY + "px";
-            blast.elem.style.width = blast.size + "px";
-            blast.elem.style.height = blast.size + "px";
-        }
-        if (this.players[num].blastCounter > 0) {
-            this.players[num].blastCounter -= 1;
-        }
-    }
-
-    private blast(x: number, y: number, playerIndex: number) {
-        this.blasts[playerIndex].posX = x;
-        this.blasts[playerIndex].posY = y;
-        this.blasts[playerIndex].size = this.config.playerSize;
-        this.blasts[playerIndex].opacity = 0.5;
-        this.players.forEach((player) => {
-            const distance = Math.sqrt(Math.pow(x - player.posX, 2) + Math.pow(y - player.posY, 2));
+    private blast(player: Player) {
```

Comment:
I don't think there's any performance penalty (or very little) for passing the player object into this function